project_task_materials_stock/views/project_view.xml

```diff
@@ -14,46 +13,40 @@
-        <field name="stage_id" position="before">
-          <button name="action_assign" type="object"
-                  string="Check Availability"
-                  attrs="{'invisible':[('stock_state','!=','confirmed')]}"
-                  class="oe_highlight"/>
-          <button name="action_done" type="object" string="Transfer Materials"
-                  attrs="{'invisible':[('stock_state','!=','assigned')]}"
-                  class="oe_highlight"/>
-        </field>
-        <field name="material_ids" position="replace"/>
-        <field position="after" name="work_ids">
-          <field name="stock_state" invisible="1"/>
-          <field name="consume_material" invisible="1"/>
-          <group string="Materials consumed"
-                 groups="project.group_tasks_work_on_tasks">
-            <notebook>
-              <page string="Products">
-                <field name="material_ids"
-                       attrs="{'readonly':[('stock_state','!=', 'pending')]}">
-                  <tree string="Materials used" editable="top">
-                    <field name="product_id"
-                           domain="[('type', 'in', ['consu', 'product'])]"/>
-                    <field name="quantity"/>
-                    <field name="product_uom" groups="product.group_uos"/>
-                  </tree>
-                </field>
-              </page>
-              <page string="Stock Moves">
-                <field name="stock_move_ids"/>
-              </page>
-            </notebook>
-          </group>
+        <field name="name">project.task.form.materials</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project_task_materials.view_project_task_form_materials"/>
+        <field name="arch" type="xml">
+            <field name="stage_id" position="before">
+                <button name="action_assign" type="object"
+                        string="Check Availability"
+                        attrs="{'invisible':[('stock_state','!=','confirmed')]}"
+                        class="oe_highlight"/>
+                <button name="action_done" type="object" string="Transfer Materials"
+                        attrs="{'invisible':[('stock_state','!=','assigned')]}"
+                        class="oe_highlight"/>
+            </field>
+            <field name="material_ids" position="replace">
+                <field name="stock_state" invisible="1"/>
+                <field name="consume_material" invisible="1"/>
+                <group string="Materials consumed" groups="project.group_tasks_work_on_tasks">
+                    <notebook>
+                        <page string="Products">
+                            <field name="material_ids" attrs="{'readonly':[('stock_state','!=', 'pending')]}">
+                                <tree string="Materials used" editable="top">
+                                    <field name="product_id" domain="[('type', 'in', ['consu', 'product'])]"/>
```

Comment:
Consume action create an stock move and stock move only can be generated with products that not are services.
That's the reason.